docs/en/modules/admin/pages/components/budgets.adoc

```diff
@@ -1,130 +1,125 @@
 = Budgets
-:page-toclevels: 4
 
-The Budgets component allows people to vote by selecting projects of their choosing. Depending on the needs and normative
-of the organization, this could be configured so people have assigned a total budget (for instance, 100.000 EUR) and then
-every project has assigned a value (for instance, 10.000 EUR). Then when people vote on projects, they have a progress bar
-where they see how much remaining budget they have to allocate in this voting.
+== Understand the basics
 
-From this starting point, there could be other configurations, such as:
+The Budgets component allows participants to vote by selecting projects of their choosing. Administrators can decide the voting rules, 
+to make participants select a defined number of projects, or vote for as many project they want until reaching a certain amount. 
 
```

Comment:
I suggest to add here a sentence to explain the purpose of this component. Here is my suggestion, but i you came up with something clearer, ho ahead:

This component has been designed to organise voting in the final phase of a Participatory Budget, although it can be used for other types of processes.